manifests/p/PolarGoose/CoffeeBean/4.0/PolarGoose.CoffeeBean.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: PolarGoose.CoffeeBean
+PackageVersion: 4.0
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://github.com/PolarGoose/CoffeeBean/releases/download/v4.0/CoffeeBean.msi
+  InstallerSha256: 5A35D723D1D580BDF6A18DC3E32D2FA8040F0A36DC148E24DB0B88E3FE1906FC
+  ProductCode: '{5BB1ADB6-710D-4BD3-9815-0FB0AA5D6148}'
```

Comment:
```suggestion
  ProductCode: '{79E9FD62-8839-4929-BC92-3B3606516755}'
```
![image](https://user-images.githubusercontent.com/11600822/147878341-c8b7a0ab-e894-4dc6-99e8-1de071ead83d.png)

Reply:
If you use `YamlCreate.ps1` or wingetcreate, this might be automatically done for you.

Reply:
@ItzLevvie ,
Thank you for the remark. I used `wingetcreate` to create the manifest. The problem was that `wingetcreate` stores all installers in a cache. I needed to call `wingetcreate cache --clean` and then it generated the correct values.

Could you please check my changes again?